README.md

```diff
@@ -1,4 +1,5 @@
 [comment]: # (https://shields.io/ is a good source of these)
+[![Cirrus-CI]("https://api.cirrus-ci.com/github/SciTools-incubator/iris-ugrid.svg?branch=master")](https://cirrus-ci.com/github/SciTools-incubator/iris-ugrid)
```

Comment:
Do we need the quotation marks around https://....branch=master? The black line below doesn't have them. Might be a red herring but on the GitHub PR rich diff rendering of the new version doesn't display the cirrus-ci badge correctly

Reply:
I'll take care of that. But I suspect the bad rendering is because there hasn't yet been a Cirrus run against `master`

Reply:
9242f3f16c2adf41c78c6a933c7d98668f677f75